app/src/main/res/values/strings.xml

```diff
@@ -8,72 +8,53 @@
+    <string name="LearningSettings_presentationButton">PRESENTATION</string>
+    <string name="LearningSettings_representationButton">REPRESENTATION</string>
+    <string name="LearningSettings_evaluationButton">EVALUATION</string>
+    <string name="LearningSettings_tvLearningModeText">LEARNING MODE</string>
 
-    <string name="info">Info</string>
-    <string name="modify_categories">Modify name and categories</string>
-    <string name="add_new_picture">Add new picture</string>
-    <string name="delete_dataset">Delete dataset</string>
-    <string name="enter_dataset_name">Enter dataset name</string>
-    <string name="ds_name">Dataset name</string>
-    <string name="imagetobedisplayed">image to be displayed</string>
-    <string name="DataOverview_CreateButton">Create dataset</string>
+    <string name="MetadataEditing_infoTitle">Info</string>
+    <string name="MetadataEditing_modifyCategoriesButton">Modify name and categories</string>
+    <string name="DisplayDataset_addNewPictureButton">Add new picture</string>
+    <string name="DisplayDataset_deleteDatasetButton">Delete dataset</string>
+    <string name="MetadataEditing_enterDatasetNameHintText">Enter dataset name</string>
+    <string name="DisplayDataset_datasetNameText">Dataset name</string>
+    <string name="DisplayImage_imageTobeDisplayed">image to be displayed</string>
+    <string name="DatasetsOverview_createButton">Create dataset</string>
     <string name="DatasetsOverview_Title">Datasets Overview</string>
-    <string name="deletingDataset">Deleting dataset</string>
-    <string name="deleteConfirmationMessage">Are you sure you want to delete this dataset?</string>
-    <string name="metadataEditingInfo"> General information :\n\n
-        - A category is a word representing a type of object (Ex: Fork).\n\n
-        - Every picture belongs to a category of the dataset.\n\n
-    Possible actions :\n\n
-    - You can add a new category and name it as you wish by clicking the "Add new category"
-    button.\n\n
-    - You can delete a category by clicking on the red trashcan icon.\n\n
-    - You can modify the name of the dataset by clicking on the name of the dataset on the top of
-    the screen. \n\n
-    - Once you have finished everything you wanted to do, you can click on the "Save data" button to
-    save your modifications.</string>
-    <string name="displayImagesetInfo">
-        General information :\n\n
-        - Here are displayed all the pictures of the selected category.\n\n
-        - A representative picture is a picture that is used to represent the category in the dataset.
-        Such a picture is not part of the pictures of the category.\n\n
-        Possible actions :\n\n
-        - To delete pictures : click longer on the picture you want to delete, select all the pictures to delete and press the trashcan icon.\n\n
-        - To set an picture as representative of the category : click longer on the picture and select the star icon.</string>
```

Comment:
where did those go?

Reply:
I just removed these when resolving the conflicts, you can see those same resources refactored just above

Reply:
Ok found it. It's different than the original tho, why is that? And where is the information on deleting a category and renaming a dataset?

Reply:
Looking at the view in context, these instructions seem redundant. Is that the idea?
![Screenshot_20210603-225218_One UI Home.jpg](https://user-images.githubusercontent.com/15657735/120710432-6ad36000-c4be-11eb-9936-491843c46ef6.jpg)

Reply:
Yes that was the idea initially but I think it's Ok to leave them for first time users so I ll put them back -- and readd the info that I mistakenly removed